### PR TITLE
Improve README, make it shorter and more functional

### PR DESCRIPTION
**Pull request by qrutyy**
#### Improve README by making it shorter -> make it more understandable.

Move the tech features from the README to WIKI. Embellished a lil)

**TODO:**
- refactor the code after the review
- add everything to WIKI
- make a new release

### CODE REVIEW

**Review by shakareem — APPROVED** [review]

File: README.md [hunk 1]
```diff
@@ -1,19 +1,23 @@
...
+- Store graphs anywhere you want: SQLite, Neo4J, JSON (WIP) (see [wiki])
 - Drag, zoom, replace nodes
 
-![mainscreen_screenshot](https://github.com/spbu-coding-2023/graphs-graph-2/assets/64466788/9c708a11-dc6e-4cf9-a848-44f21dec7a37)
+![alt text][main_screen_image]
 
 ## Usage
 
-Firstly - clone the last release/version of our app. Since our app uses Gradle build system - you can use these commands:
+Firstly - clone the last release/version of our app. Since our app uses Gradle build system - you can use this commands:
```
Comment (spisladqo) [inline README.md]: These commands was correct

Comment (qrutyy) [reply README.md]: thx!😃

File: README.md [hunk 2]
```diff
@@ -1,15 +1,19 @@
...
 
 ## Features
 
-- Support 4 types of graphs (see *)
-- Use all in all - 9 algorithms
-- Store graphs anywhere you want: SQLite, Neo4J, JSON
+### All in all:
+- Support 4 types of graphs
+- Use all in all - 9 algorithms (SCC, bridges, shortest paths, cycles, layout, communities, etc. see [wiki])
+- Store graphs anywhere you want: SQLite, Neo4J, JSON (WIP) (see [wiki])
```
Comment (spisladqo) [inline README.md]: I think this paragraph's name is redundant, cause these are just features like stated above, and why repeat **all in all** twice?

Comment (qrutyy) [reply README.md]: resolved

**Review by spisladqo — APPROVED** [review]

**Pull request merged**